source/app.d

```diff
@@ -2796,28 +2797,36 @@ void assemblyLine() {
 }
 
 void assemblyFile(string filename) {
-	filename = filename.defaultExtension("asx");
-	if (getOption('p'))
-		filename = absolutePath(filename);
-	File stream = openInputFile(filename);
-	scope (exit) stream.close();
+	immutable(ubyte)[] source = sourceFiles.require(filename, {
```

Comment:
https://app.travis-ci.com/github/pfusik/xasm/jobs/589544515

    source/app.d:2800:41: error: no property 'require' for type 'immutable(ubyte)[][string]'
      immutable(ubyte)[] source = sourceFiles.require(filename, {

Reply:
https://travis-ci.community/t/could-someone-please-upgrade-the-gdc-travis-use/4533 ?

Reply:
Yes. 4.8.5 is older than Vasco. It's not unreasonable to expect at least GDC 10 nowadays. Probably replacing Travis with some other CI will be more practical than waiting for Travis update.

Reply:
Still better than travis